resources/assets/uad_lists.json

```
@@ -19145,7 +19145,7 @@
   {
     "id": "com.sonyericsson.warrantytime",
     "list": "Oem",
-    "description": "Lets you see some info about your warranty and how long it will last.",
+    "description": "Let's you see some info about your warranty and how long it will last.",
```

```suggestion
    "description": "Lets you see some info about your warranty and how long it will last.",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.